bears/haml/HamlLintBear.py

```diff
@@ -0,0 +1,69 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.GemRequirement import GemRequirement
+import yaml
+
+
+@linter(executable='haml-lint',
+        output_format='regex',
+        output_regex=r'(?P<filename>\S+):(?P<line>\d+) \[(?P<severity>W|E)\] '
+                     r'(?P<message>.*)',
+        severity_map={'E': RESULT_SEVERITY.MAJOR,
```

Comment:
Please remove the `filename` capturing group because that is relevant for global bears only.